### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Use a connection string_Attach with Connection String_ScreenReader: JAWS is not announcing the error messages as soon as it is thrown.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/AzureStorageExplorer#3468]
**GitHub Tags:** #C&AI_Aug2020_BMAzure Tools Storage Explorer; #A11yMAS; #A11YTCS; #DesktopApp; #Azure Tools Storage Explorer; #WCAG1.3.1; #Win32; #A11ySev2; #Benchmark; #JAWS;#NVDA;#WCAG3.3.1,#FTP;#STP

### Environment Details:
Application Name: Azure Tool Storage Explorer
Application Version: 1.14.2
Windows Version: Windows 10
Screen Reader : JAWS(2020.2006.12ILM),NVDA(2020.2)

### Repro Steps:
1. Launch Storage Explorer
2. Navigate to "Open Connect Dialog" pane present on left side of screen and hit enter.
3. Select "Use a connection string" radio button and hit enter on Next button.
4. "Attach with Connection string" screen gets opened
5. Start JAWS and navigate using tab key.
6. Navigate till "Connection string" edit box using tab key and add some text, it will throw an error message if the text is wrong.
7. Observe the announcement by JAWS for the error message.

### Actual:
The JAWS is not announcing the error message thrown for the "Connection string" edit box that is "The connection string is missing an account key or SAS token", as soon as it is thrown.
**This issue is also observed with the NVDA**
**Similar issue is also observed with following scenarios:**

#### Use a SAS URL-Attach with SAS URL
1. Launch Storage Explorer
2. Navigate to "Open Connect Dialog" pane present on left side of screen and hit enter.
3. Select "Use a SAS URL" radio button and hit enter on Next button.
4. "Attach with SAS URL" screen gets opened.
5. Start JAWS and navigate using tab key.
6. Navigate till "URL" edit box using tab key and add some text, it will throw an error message if the text is wrong.
7. Observe the announcement by JAWS for the error message.

### Expected:
The JAWS should announce the error message thrown for the "Connection string" edit box. 

### User Impact:
Screen reader users won't able to come to know that there is a error message thrown on the screen.

### Attachment:
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-shwarh_microsoft_com/EY0dY9zbavZCjVFqF4ZXpfkBXGv_r-dXDAyD8KgYb4tIDg?e=DR6ikD

**Comment by 12-shweta** [comment]
@MRayermannMSFT 
Here the issue is that when we edit the text in the "Connection string" edit box, there is a one error message appears on the screen stating  "The connection string is missing an account key or SAS token" but this error message is not announced by the JAWS but NVDA does.
So here the expected is that the JAWS should announce the status message as soon as it is thrown.

**Comment by craxal** [comment]
@12-shweta I am unable to repro the issue. When I start typing in the Connection String input, I immediately hear “alert” from the screen reader. Granted, if I keep typing rapidly, I won’t hear the actual error message, but I still hear the “alert” part. I observe this same behavior in NVDA and JAWS.

**Comment by 12-shweta** [comment]
Verified the bug in the version 1.16.0 - dev
The issue is no longer reproduce and hence closing the bug.
Attachment:
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-shwarh_microsoft_com/EXDAkMfJk_BIgHLEl1W3ftEBphwlCMSiV8_-8Y4T26o3lg?e=Kh0tDF

**Comment by 12-shweta** [comment]
Github Tags:#Closed;
